src/sidebar/test/media-embedder-test.js

```diff
@@ -90,6 +90,24 @@ describe('media-embedder', function () {
     });
   });
 
+  it('replaces internet archive links with iframes', function () {
+    var urls = [
+      'https://archive.org/details/WHDH_20151121_043400_The_Tonight_Show_Starring_Jimmy_Fallon/start/360/end/420.3',
+      'https://archive.org/embed/WHDH_20151121_043400_The_Tonight_Show_Starring_Jimmy_Fallon?start=360&end=420.3',
+    ];
+    urls.forEach(function (url) {
+      var element = domElement('<a href="' + url + '">' + url + '</a>');
+
+      mediaEmbedder.replaceLinksWithEmbeds(element);
+
+      assert.equal(element.childElementCount, 1);
```

Comment:
Since we only have one domElement, this will be 1. I think its ok to leave this assertion out.